src/game/mario.c

```diff
@@ -530,6 +535,8 @@ s32 mario_facing_downhill(struct MarioState *m, s32 turnYaw) {
  */
 u32 mario_floor_is_slippery(struct MarioState *m) {
     f32 normY;
+    if (m->floor->type == SURFACE_SUPER_SLIPPERY)
+        return TRUE;
```

Comment:
why not just combine all these checks as an OR into the check below that checks for slide terrain? would be a lot neater IMO